esp/src/eclwatch/ESPRequest.js

```diff
@@ -95,7 +95,7 @@ define([
             return retVal.then(function (response) {
                 params.load(response);
                 return response;
-            }).catch(function(e) {
+            }).catch(function(error) {
```

Comment:
Is this part of the webpack improvements? (I know we moved away from this argument naming convention)

Reply:
Its fixing the linting errors